prow/cluster/resources/gatekeeper-constraints/workloads/signifySecretTrustedUsage.yaml

```diff
@@ -19,13 +19,19 @@ spec:
         command:
           - /tools/entrypoint
         args: []
-        entrypoint_options: '^{.*"args":\["\/image-builder","--name=.+","--config=\/config\/kaniko-build-config\.yaml","--context=\.","--dockerfile=.+"\],"container_name":"test",.*}$'
+        entrypoint_options: '^{.*"args":\["\/image-builder","--name=.+","--config=\/config\/kaniko-build-config\.yaml","--context=.+","--dockerfile=.+"\],"container_name":"test",.*}$'
```

Comment:
I think this dot here represent a current directory. It's not part of any regex.

Reply:
One of the jobs uses different directory here, I could leave it as it is or split into two trustedUsages

(`pull-function-runtimes-python39-build` uses `--context=components/function-runtimes/python39`)

Reply:
Let's keep your version.